lib/tests-transform-filter.js

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
-var Filter = require('broccoli-persistent-filter');
-var transform = require('./transform-qunit-assertions');
-var path = require('path');
-var minimatch = require('minimatch');
+import { call, prototype } from 'broccoli-persistent-filter';
```

Comment:
Is it possible to decouple this refactor from the upgrade and have it in a separate PR?